File: MCServerWebWrapper.Server/ClientApp/src/app/models/cpu-data.ts
export class CpuData {
  cpuLogs: number[];
  xValues: number[];

  constructor() {
    this.cpuLogs = [];
    this.xValues = [];
    for (var i = 1; i <= 60; i++) {
      this.xValues.push(i * 7);
    }
  }

  public addData(cpuUsage: number) {
    if (this.cpuLogs.length >= 60) {
      this.cpuLogs.pop();
    }
    this.cpuLogs.unshift((100 - cpuUsage) * 2);
    return this.getString();
  }

  getString() {
    var arr = [];
    for (var i = 0; i < this.cpuLogs.length; i++) {
      arr.push(`${this.xValues[i]},${this.cpuLogs[i]}`);
    }
    return arr.join(" ");
  }
}

File: MCServerWebWrapper.Server/Models/CpuData.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace MCServerWebWrapper.Server.Models
{
	public class CpuData
	{
		public List<int> CpuLogs { get; set; }
		private List<int> xValues { get; set; }

		public CpuData()
		{
			CpuLogs = new List<int>();
			xValues = new List<int>();
			for (var i = 0; i < 60; i++)
			{
				xValues.Add(i * 7);
			}
		}

		public string AddDataAndGetString(int cpuUsage)
		{
			if (CpuLogs.Count >= 60)
			{
				CpuLogs.RemoveAt(CpuLogs.Count - 1);
			}
			CpuLogs.Insert(0, (100 - cpuUsage) * 2);
			return GetString();
		}

		public string GetString()
		{
			var coordsList = new List<string>();
			coordsList.Insert(0, "0,200");
			for (var i = 0; i < CpuLogs.Count; i++)
			{
				coordsList.Add($"{xValues[i]},{CpuLogs[i]}");
				if (i == CpuLogs.Count - 1)
				{
					coordsList.Add($"{xValues[i]},200");
				}
			}
			return String.Join(" ", coordsList);
		}
	}
}
